--- contrib/poco/Foundation/src/Format.cpp	2026-05-28 17:56:55.626173657 +0200
+++ contrib/poco-cmake/patches/Format.cpp.new	2026-05-28 18:43:24.819681826 +0200
@@ -228,7 +228,7 @@
 std::string format(const std::string& fmt, const Any& value)
 {
 	std::string result;
-	format(result, fmt, value);
+	Poco::format(result, fmt, value);
 	return result;
 }
 
@@ -236,7 +236,7 @@
 std::string format(const std::string& fmt, const Any& value1, const Any& value2)
 {
 	std::string result;
-	format(result, fmt, value1, value2);
+	Poco::format(result, fmt, value1, value2);
 	return result;
 }
 
@@ -244,7 +244,7 @@
 std::string format(const std::string& fmt, const Any& value1, const Any& value2, const Any& value3)
 {
 	std::string result;
-	format(result, fmt, value1, value2, value3);
+	Poco::format(result, fmt, value1, value2, value3);
 	return result;
 }
 
@@ -252,7 +252,7 @@
 std::string format(const std::string& fmt, const Any& value1, const Any& value2, const Any& value3, const Any& value4)
 {
 	std::string result;
-	format(result, fmt, value1, value2, value3, value4);
+	Poco::format(result, fmt, value1, value2, value3, value4);
 	return result;
 }
 
@@ -260,7 +260,7 @@
 std::string format(const std::string& fmt, const Any& value1, const Any& value2, const Any& value3, const Any& value4, const Any& value5)
 {
 	std::string result;
-	format(result, fmt, value1, value2, value3, value4, value5);
+	Poco::format(result, fmt, value1, value2, value3, value4, value5);
 	return result;
 }
 
@@ -268,7 +268,7 @@
 std::string format(const std::string& fmt, const Any& value1, const Any& value2, const Any& value3, const Any& value4, const Any& value5, const Any& value6)
 {
 	std::string result;
-	format(result, fmt, value1, value2, value3, value4, value5, value6);
+	Poco::format(result, fmt, value1, value2, value3, value4, value5, value6);
 	return result;
 }
 
@@ -276,7 +276,7 @@
 std::string format(const std::string& fmt, const Any& value1, const Any& value2, const Any& value3, const Any& value4, const Any& value5, const Any& value6, const Any& value7)
 {
 	std::string result;
-	format(result, fmt, value1, value2, value3, value4, value5, value6, value7);
+	Poco::format(result, fmt, value1, value2, value3, value4, value5, value6, value7);
 	return result;
 }
 
@@ -284,7 +284,7 @@
 std::string format(const std::string& fmt, const Any& value1, const Any& value2, const Any& value3, const Any& value4, const Any& value5, const Any& value6, const Any& value7, const Any& value8)
 {
 	std::string result;
-	format(result, fmt, value1, value2, value3, value4, value5, value6, value7, value8);
+	Poco::format(result, fmt, value1, value2, value3, value4, value5, value6, value7, value8);
 	return result;
 }
 
@@ -292,7 +292,7 @@
 std::string format(const std::string& fmt, const Any& value1, const Any& value2, const Any& value3, const Any& value4, const Any& value5, const Any& value6, const Any& value7, const Any& value8, const Any& value9)
 {
 	std::string result;
-	format(result, fmt, value1, value2, value3, value4, value5, value6, value7, value8, value9);
+	Poco::format(result, fmt, value1, value2, value3, value4, value5, value6, value7, value8, value9);
 	return result;
 }
 
@@ -300,7 +300,7 @@
 std::string format(const std::string& fmt, const Any& value1, const Any& value2, const Any& value3, const Any& value4, const Any& value5, const Any& value6, const Any& value7, const Any& value8, const Any& value9, const Any& value10)
 {
 	std::string result;
-	format(result, fmt, value1, value2, value3, value4, value5, value6, value7, value8, value9, value10);
+	Poco::format(result, fmt, value1, value2, value3, value4, value5, value6, value7, value8, value9, value10);
 	return result;
 }
 
@@ -309,7 +309,7 @@
 {
 	std::vector<Any> args;
 	args.push_back(value);
-	format(result, fmt, args);
+	Poco::format(result, fmt, args);
 }
 
 
@@ -318,7 +318,7 @@
 	std::vector<Any> args;
 	args.push_back(value1);
 	args.push_back(value2);
-	format(result, fmt, args);
+	Poco::format(result, fmt, args);
 }
 
 
@@ -328,7 +328,7 @@
 	args.push_back(value1);
 	args.push_back(value2);
 	args.push_back(value3);
-	format(result, fmt, args);
+	Poco::format(result, fmt, args);
 }
 
 
@@ -339,7 +339,7 @@
 	args.push_back(value2);
 	args.push_back(value3);
 	args.push_back(value4);
-	format(result, fmt, args);
+	Poco::format(result, fmt, args);
 }
 
 
@@ -351,7 +351,7 @@
 	args.push_back(value3);
 	args.push_back(value4);
 	args.push_back(value5);
-	format(result, fmt, args);
+	Poco::format(result, fmt, args);
 }
 
 
@@ -364,7 +364,7 @@
 	args.push_back(value4);
 	args.push_back(value5);
 	args.push_back(value6);
-	format(result, fmt, args);
+	Poco::format(result, fmt, args);
 }
 
 
@@ -378,7 +378,7 @@
 	args.push_back(value5);
 	args.push_back(value6);
 	args.push_back(value7);
-	format(result, fmt, args);
+	Poco::format(result, fmt, args);
 }
 
 
@@ -393,7 +393,7 @@
 	args.push_back(value6);
 	args.push_back(value7);
 	args.push_back(value8);
-	format(result, fmt, args);
+	Poco::format(result, fmt, args);
 }
 
 
@@ -409,7 +409,7 @@
 	args.push_back(value7);
 	args.push_back(value8);
 	args.push_back(value9);
-	format(result, fmt, args);
+	Poco::format(result, fmt, args);
 }
 
 
@@ -426,7 +426,7 @@
 	args.push_back(value8);
 	args.push_back(value9);
 	args.push_back(value10);
-	format(result, fmt, args);
+	Poco::format(result, fmt, args);
 }
 
 
